requirements-check.txt

```diff
@@ -0,0 +1,3 @@
+# Local development
```

Comment:
Not sure the purpose of this file, how it is supposed to be used? Should we mention that somewhere?

Reply:
I was not sure if I should add mypy and precommit to requirements-dev, that would be the easiest and no extra changes needed.

Or I could add more on that filei. Contributing.